From 813c90fcf4750b72747dc0f6d4c800ffb0578c96 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 2 Jun 2020 21:35:54 +0200
Subject: [PATCH 1/1] config: remove CONFIG_ARMV8_SWITCH_TO_EL1 from
 whitelist.txt

Since commit 208bdaf2ae5f ("vexpress/aemv8a: drop
CONFIG_ARMV8_SWITCH_TO_EL1") no board uses CONFIG_ARMV8_SWITCH_TO_EL1. So
we should remove this setting from whitelist.txt. If anybody should ever
need it again, he should add a Kconfig option.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 scripts/config_whitelist.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/config_whitelist.txt b/scripts/config_whitelist.txt
index fe9a1971cc..15b77c981e 100644
--- a/scripts/config_whitelist.txt
+++ b/scripts/config_whitelist.txt
@@ -49,7 +49,6 @@ CONFIG_ARMV7_PSCI_1_0
 CONFIG_ARMV7_SECURE_BASE
 CONFIG_ARMV7_SECURE_MAX_SIZE
 CONFIG_ARMV7_SECURE_RESERVE_SIZE
-CONFIG_ARMV8_SWITCH_TO_EL1
 CONFIG_ARM_ARCH_CP15_ERRATA
 CONFIG_ARM_DCC
 CONFIG_ARM_FREQ
-- 
2.26.2

